renovation_core_graphql/translation/types/root.graphql

```diff
@@ -1,6 +1,7 @@
 extend type Mutation {
     addTranslation(language: String!, source_text: String!, translated_text: String!,
       context: String, doctype: String, docname: String, docfield: String): Translation!
```

Comment:
I do recommend refactoring this mutation `AddTranslationInput`. 
Maybe in this PR, or a separate one.

Reply:
done 👍🏼